SEP/SEP-0007/sep-0007.md

```diff
@@ -58,16 +58,16 @@ SPARQL Exists Community group mailing list [public-sparql-exists/2016Jul/0014](h
 * [Substitution affects disconnected variables.](#issue-5-substitution-affects-disconnected-variables)
 
 ### Issue 1: Some uses of `EXISTS` are not defined during evaluation
-The evaluation process in the specificiation is defined for graph patterns but
-there are situations where the evaluation is of an alegbra form not listed.
+The evaluation process in the specification is defined for graph patterns, but
+there are situations where the evaluation is of an unlisted algebra form.
 
-For example:
+For example —
 
 ```sparql
 FILTER EXISTS { SELECT ?y { ?y :q :c } }
 ```
 
-and
+— and —
```

Comment:
The PR has a lot of em-dashes used for colons and also added them in places I find unnecessary such as the continuing text (it's not dialogue).

Please change it back to colons. It's my style to use colons.

The styleguide on the web I've checked (if they mention it at all) use a colon when the list is not inline nor first. em-dash is for breaking up running text lists, not bulleted lists or code blocks.

(Styleguides are like standards - so many to choose from)

Reply:
fwiw, the [W3 Style Guide](https://www.w3.org/Guide/manual-of-style/) says to refer to The Chicago Manual of Style and/or The Gregg Reference Manual, neither of which is freely available on the web (so no citation links).

These two agree that the independent clause preceding a colon _should_ be able to stand on its own, i.e., be a complete sentence. This is typically most easily accomplished by adding a phrase like `the following` to an existing leading fragment.

Neither Manual has good advice for this kind of flowing text which, while not a dialog, _is_ a continuous sentence, as written. The intervening line of just `and` is more than a trifle awkward to my eye.

This might be a use for an unordered list (even though there are only two items), led by `Examples include:`.  I _believe_ I've found Markdown that will properly put a codeblock as a bullet item (instead of below/within it).  If you might be OK with that, I'll edit it into this PR.